example/README.md

```diff
@@ -283,36 +284,37 @@ Firstly, we need to create a **resource.** A resource represents a group of serv
     Result:
 
     [
-      "echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77"
+      "5pYpWB3ozi6fi1EjNs9X5kE156aA6iLECxTuVdJgUaLB"
     ]
     ```
 
-   So the resource id is `echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77`. Please note that the resource id might be different in your case.
+   So the resource id is `5pYpWB3ozi6fi1EjNs9X5kE156aA6iLECxTuVdJgUaLB`. Please note that the resource id might be different in your case.
 
-    Using the resource id we can access any registered provider of the resource. There can be more registered echo services on different peers that we can use transparently.
+    Using the resource id we can access any registered service. There can be more registered echo services on different peers that we can use transparently.
 
 3. This [code](src/aqua/main.aqua#L28) registers deployed service by given `resource_id`:
 
     ```rust
-    func registerService(resource_id: string) -> bool:
+    func registerService(resource_id: string) -> *bool:
```

Comment:
in `resources-api.aqua` shouldn't `unregisterService` be `unregisterResource` as the tombstone is over resource_id

Reply:
Nope, it's `unregisterService`, peer_id is passed implicitly